### PR TITLE
Add aucdi mapping update for new AUCDI R1 elements: Last Updated

### PR DESCRIPTION
**Pull request by dt-r**
Updated for AU Core R1 reconciliation of new AUCDI R1 element Last Updated, as decided in [2024-07-25 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=256508535):
* Replaced 'TBD' with '-'
* Updated comments for Last Updated with link to future use section
* Additionally, improved comments format for Medication Use Statement and added link to Medicine Information Guidance.
* Added Last Updated to Future of AU Core
* Added change log for the new aucdi date/time element mappings

### CODE REVIEW

File: input/pagecontent/future.md [hunk 1]
```diff
@@ -35,4 +35,7 @@ The approach to develop and mature AU Core is outlined below making reference to
 The following items are under consideration to add to AU Core.
 
 #### AU Core MedicationStatement 
-Development and testing of AU Core MedicationStatement profile to support AUCDI Medication Use Statement is planned for the AU Core R2 scope.
\ No newline at end of file
+Development and testing of AU Core MedicationStatement profile to support AUCDI Medication Use Statement is planned for the AU Core R2 scope.
+
+#### Last Updated 
+AUCDI element Last Updated is intended to to reflect when a data group was last updated, capturing clinical relevance and currency. A known and implemented solution in other jurisdictions for similar requirements is the use of the Provenance resource. An alternative to the use of Provenance may be the definition of custom extensions. Future versions of AU Core will develop and test approaches to addressing Last Updated.
```
Comment (dbojicic) [inline input/pagecontent/future.md:41]: Remove duplicated word 'to'

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 This change log documents the significant updates and resolutions implemented from version 0.3.0-ballot to the upcoming August ballot.
 
 #### Changes in this version
+- added AUCDI element mapping Date/Time of Measurement and Date/Time of Observation, and added future consideration note for Last Updated
```
Comment (dbojicic) [inline input/pagecontent/changes.md:9]: The sentence should start with an uppercase letter, please change

File: input/pagecontent/aucdi.md [hunk 3]
```diff
@@ -157,64 +157,57 @@ Column attribute descriptions are as follows:
...
+    <td>-</td>
+    <td>This AUCDI element is a <a href="future.html#future-candidate-requirements-under-consideration">Future Candidate Requirement Under Consideration</a>. Future versions of AU Core will develop and test approaches to addressing Last Updated.</td>
   </tr>
   <tr>
     <td rowspan="9">Medication use statement</td>
     <td colspan="2">Medication name</td>
     <td>AU Core MedicationStatement | <a href="StructureDefinition-au-core-medication.html">AU Core Medication</a></td>
     <td>MedicationStatement.medication[x] | Medication.code</td>
-    <td>Development and testing of AU Core MedicationStatement profile is planned for AU Core R2.</td>
+    <td rowspan="8"><p>Development and testing of AU Core MedicationStatement profile is planned for AU Core R2.</p><p>See <a href="medicine-information.html">Medicine Information Guidance</a>.</p></td>
```
Comment (dbojicic) [inline input/pagecontent/aucdi.md:169]: Suggest changing the hyperlink text to match the page title for consistency.

**Review by dbojicic — CHANGES_REQUESTED** [review]
See inline comments.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/changes.md:9]: fixed

File: input/pagecontent/aucdi.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/aucdi.md:169]: The page title is Medicine Information. But If you are concerned on having the word guidance inside the hyperlink I'll change to that for now. I think it's better with it in.

**Pull request merged**